views/content/content.html

```diff
@@ -15,25 +15,24 @@ <h2 class="base--h2">Revisions</h2>
 
 {% if content %}
 
-  <table class="table--style">
+  <table class="base--table">
     <thead class="base--thead">
       <tr class="base--tr">
         <th class="base--th">revision</th>
         <th class="base--th">date</th>
         <th class="base--th">{{config.base}}</th>
-        <th class="base--th">{{config.actions.edit}}</th>
-        <th class="base--th">{{config.actions.approve}}</th>
+        <th class="base--th" colspan="2">Actions</th>
       </tr>
     </thead>
     <tbody class="base--tbody">
     {% for c in content %}
       {% if c.value %}
         <tr class="base--tr">
-          <td class="base--td">{{c.revision}}</td>
+          <td data-th="Revision" class="base--td">{{c.revision}}</td>
           <td data-th="date" class="base--td">{{c.created}}</td>
```

Comment:
These dates are coming out yuge strings. Got a date-cleanup idea? We should be able to choose a way with a filter. We'd prol want to mimic this format for dates in other places (like sunrise/sunset)